dev/sg/sg_lint.go

```diff
@@ -97,6 +105,11 @@ sg lint --help
 			return errors.Wrap(err, "repo.GetState")
 		}
 
+		if !lintNoFormatCheck.Get(cmd) {
```

Comment:
```suggestion
        doFormatCheck = !lintNoFormatCheck.Get(cmd)
		if doFormatCheck {
```
more a nit than anything else